docs/pages/router/routing/groups.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Groups
+---
+
+You can prevent a segment from showing in the URL by using the group syntax `()`.
+
+- `app/root/home.js` -- Matches `/root/home`.
+- `app/(root)/home.js` -- Matches `/home`.
+
+This is useful for adding layouts without adding additional segments to the URL.
+
+### Shared Routes
+
+To match the same URL with different layouts, use _groups_ with overlapping child routes.
+
+This pattern is very common in native apps. For example, in Twitter, a profile can be viewed in every tab (home, search, profile). There should only be one URL needed to access this route however.
+
+```bash title="File System"
+app/
+  _layout.js # Tab bar
+  (home)
+    _layout.js # Header
+    [user].js # Matches `/baconbrix`
+  (search)
+    _layout.js # Header with search bar
+    [user].js # Also matches `/baconbrix`
+  (profile)
+    _layout.js # Folding profile header
+    [user].js # Also matches `/baconbrix`
+```
+
```

Comment:
Let's use `FileTree` component to render the project structure instead of ASCII art.